js/src/widgets/contextControls.js

```diff
@@ -168,6 +200,15 @@
                                    '<a class="hud-control draw-tool">',
                                    '<input type="text" class="fillColorPicker"/>',
                                    '</a>',
+                                   '{{#if showStrokeStyle}}',
+                                   '<a href="javascript:;" class="mirador-btn draw-tool mirador-line-type" role="button" aria-label="Change Line Type"><i class="material-icons solid">create</i>',
+                                   '<ul class="dropdown type-list">',
+                                   '<li><i class="fa solid"></i> solid</li>',
```

Comment:
The text used in these `<li>` elements must use i18next and the translation files